rootston/config.c

```diff
@@ -389,6 +392,7 @@ struct roots_config *roots_config_create_from_args(int argc, char *argv[]) {
 	}
 
 	config->xwayland = true;
+	config->xwayland_lazy = false;
```

Comment:
Technically redundant with calloc, but I actually think this should default to true if it works well.

Reply:
This works really well (at least on my machine), so I think this should be the default too, at least in rootston. @SirCmpwn thoughts?

Reply:
Oh, one last thing: can you update the config file example explaining that `lazy` exists and describing what it does?